easybuild/easyconfigs/c/cryoDRGN/cryoDRGN-0.3.4-foss-2021a-CUDA-11.3.1.eb

```diff
@@ -0,0 +1,76 @@
+easyblock = 'PythonBundle'
+
+name = 'cryoDRGN'
+version = '0.3.4'
+versionsuffix = '-CUDA-11.3.1'
+
+homepage = 'https://cb.csail.mit.edu/cb/cryodrgn/'
+description = """cryoDRGN: Deep Reconstructing Generative Networks for cryo-EM heterogeneous
+ reconstruction.
+ CryoDRGN is a neural network based algorithm for heterogeneous cryo-EM reconstruction. In
+ particular, the method models a continuous distribution over 3D structures by using a neural
+ network based representation for the volume.
+"""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('SciPy-bundle', '2021.05'),
+    ('IPython', '7.25.0'),
+    ('JupyterLab', '3.0.16'),
+    ('PyTorch', '1.10.0', versionsuffix),
```

Comment:
Apparently cryoDRGN doesn't work on top of PyTorch 1.10, see https://github.com/zhonge/cryodrgn/issues/66 and https://github.com/zhonge/cryodrgn/issues/81

The issue is fixed in cryoDRGN v0.3.5, see let's bump to that, see #15609